frameworks/cassandra/docs/upgrade.md

```diff
@@ -0,0 +1,23 @@
+---
+post_title: Upgrade
+menu_order: 130
+feature_maturity: preview
+enterprise: 'no'
+---
+
+# Overview
+We support upgrade/rollback between adjacent versions only. Concretely, to upgrade from version 2 to version 4, you must upgrade from 2 -> 3, then from 3 -> 4.
+
+# Upgrade Instructions
```

Comment:
These instructions should be removed, as they're not replaced by the new upgrade feature.  You can leave the Overview section.